packages/desktop-client/src/style/themes/light.ts

```diff
@@ -124,11 +124,13 @@ export const formInputBackgroundSelection = colorPalette.purple500;
 export const formInputBorder = colorPalette.navy300;
 export const formInputTextReadOnlySelection = colorPalette.navy50;
 export const formInputBorderSelected = colorPalette.purple500;
+export const altformInputBorderSelected = colorPalette.blue500;
```

Comment:
Where is the non-`alt` version of this color used? I would be in favor of changing the input to have a purple border when focused, or change the regular color back to blue and using that.

Reply:
I'm fine with either way. I was actually going to ask about this. Does anyone have a preference? 

@MatissJanis do you care if it's a blue border or a purpule border when in focus?

Reply:
I think purple makes more sense as it's our primary color.